tests/integration/test_sns.py

```diff
@@ -11,78 +11,75 @@
 TEST_TOPIC_NAME = 'TestTopic_snsTest'
 TEST_QUEUE_NAME = 'TestQueue_snsTest'
 
-TEST_TOPIC_RAW_NAME = 'TestTopic_snsTest_raw'
-TEST_QUEUE_RAW_NAME = 'TestQueue_snsTest_raw'
-
 
 class SNSTest(unittest.TestCase):
 
-    def test_publish_unicode_chars(self):
-        sqs_client = aws_stack.connect_to_service('sqs')
-        sns_client = aws_stack.connect_to_service('sns')
+    def setUp(self):
```

Comment:
Nit: Perhaps we could use `setUpClass` and `tearDownClass` to run the init/teardown logic only once (instead of for each test method separately).

Reply:
Problem is that if I do that, I still have to remove the topic each time I think, else the subscriptions will be duplicated and it may fails the test.
WDYT ?

Reply:
Good point - I just realized `TEST_TOPIC_RAW_NAME` was removed. Looks good, thanks.